hydra_base/lib/project.py

```diff
@@ -614,22 +618,12 @@ def clone_project(project_id,
 
     return new_project.id
 
-
-def _get_project_hierarchy(project_id, user_id):
-
-    proj = JSONObject(_get_project(project_id, user_id=user_id))
-
-    project_hierarchy = [proj]
-    if proj.parent_id:
-        project_hierarchy = project_hierarchy + _get_project_hierarchy(proj.parent_id, user_id)
-
-    return project_hierarchy
-
 def get_project_hierarchy(project_id, **kwargs):
     """
-        Return a list of project-ids which represent the links in the chain up to the root project
-        [project_id, parent_id, parent_parent_id ...etc]
-        If the project has no parent, return [project_id]
+        Return a list of project JSONObjects which represent the links in the chain up to the root project
+        [project_j, parent_j, parent_parent_j ...etc]
+        If the project has no parent, return [project_j]
     """
     user_id = kwargs.get('user_id')
-    return _get_project_hierarchy(project_id, user_id)
+    hierarchy = _get_project(user_id).get_hierarchy()
```

Comment:
Tests are failing because `_get_project` has the signature...

```
_get_project(project_id, user_id, check_write=False)
```

...so there's a missing positional arg here.